internal/cli/atlas/dbusers/dbusers.go

```diff
@@ -17,11 +17,21 @@ package dbusers
 import (
 	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
 	"github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/dbusers/certs"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
 	"github.com/spf13/cobra"
 )
 
+var (
+	exampleCmd = "mongocli atlas"
+)
+
 func Builder() *cobra.Command {
 	const use = "dbusers"
+
+	if config.ToolName == config.AtlasCLI {
+		exampleCmd = "atlas"
+	}
+
```

Comment:
[q] if we're adding config dependency anyways and using this in several places, should we have a method there to fetch this command?

Reply:
I don't have a strong opinion but I think config shouldn’t contain a constant used only for the description of commands as it is not configuration-related. I will leave this comment open to see if someone has a strong opinion on this.